infrastructure/critical/modules/vhs/terraform.tf

```
@@ -1,11 +1,11 @@
 data "terraform_remote_state" "shared_infra" {
   backend = "s3"
 
-  config {
+  config = {
     role_arn = "arn:aws:iam::760097843905:role/platform-developer"
```

While you're here, update this role as well? Tiny bit of permissions hygiene we’ve been gradually applying elsewhere:

```suggestion
    role_arn = "arn:aws:iam::760097843905:role/platform-read_only"
```